### PR TITLE
Upgrade dropwizard to 0.9.3

### PR DESCRIPTION
**Pull request by ijmad**
Upgrade to DropWizard 0.9.3. This contains some newer dependencies.
Remove old/broken/unused newrelic JARs.

### CODE REVIEW

**Comment by ijmad** [comment]
rebuild

File: src/main/java/uk/gov/pay/connector/model/domain/NotificationCredentials.java [hunk 1]
```diff
@@ -12,9 +12,8 @@
 public class NotificationCredentials extends AbstractEntity {
 
     @OneToOne
-    @JsonIgnore
     @JoinColumn(name = "account_id", nullable = false)
-    @JsonManagedReference
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/connector/model/domain/NotificationCredentials.java]: Why this change?

Comment (ijmad) [reply src/main/java/uk/gov/pay/connector/model/domain/NotificationCredentials.java]: As in the commit message:

@JsonManagedDependency not matched with @JsonBackreference (as it should be) seems to upset Jackson 2.6.7 more than it upset previous versions. Remove this as the intention is not to return these fields, as @JsonIgnore is present anyway.

Comment (alexbishop1) [reply src/main/java/uk/gov/pay/connector/model/domain/NotificationCredentials.java]: Sorry, missed that.

**Review by alexbishop1 — APPROVED** [review]
Looks good 👍
